DEVELOP.md

```diff
@@ -2,12 +2,27 @@
 
 ## Documentation
 
-OUDS Android documentation is generated using [Writerside](https://www.jetbrains.com/writerside/).
-In order to modify the OUDS Android documentation or test it locally, you need to [download and install it](https://www.jetbrains.com/writerside/download/).
+OUDS Android documentation is generated using [Dokka](https://github.com/Kotlin/dokka).
 
-When installed, launch Writerside and open the `ouds-android/docs` directory.
+You can test the documentation locally by launching the `dokkaGenerate` Gradle task.
 
-In the Writerside table of content, click on "Getting started". The Writerside Preview panel is on the right with an "Open in browser" button to test
-documentation display in a browser.
+```
+./gradlew dokkaGenerate
+```
+
+When generated, open `ouds-android/docs/dokka/index.html` in your browser.
+
+## Tests
+
+Each component offered by OUDS Android has two kinds of tests: Android instrumented tests and [Paparazzi](https://github.com/cashapp/paparazzi) tests.
+
+The instrumented tests are in `ouds-android/core/src/androidTest` and allow to test interactions and behaviors.
```

Comment:
I think that `ouds-android` should be removed of the path because it's not part of the GitHub repository.